disktools.py

```diff
@@ -532,8 +532,16 @@ def partitionNumber(self, partitionDevice):
         return int(matches.group(1))
 
     # Private methods:
-    def cmdWrap(self, params):
+    def cmdWrap(self, params, checkGdisk=False):
```

Comment:
Putting sgdisk-specific checks in this library function feels a bit weird given that it is used for calling all sorts of other programs.

Reply:
yes, I'll add a `__gdiskCheck` function instead